README.md

```diff
@@ -9,26 +9,54 @@
 
 ## Usage
 
+### Singular hook
+
+Recommended for [TypeScript](#typescript)
```

Comment:
Maybe we can add a typescript example below which passes a type?

Reply:
I see you did an example below, maybe link to it?

Reply:
Don't you think this is sufficient? If you click this very link, you go to the TypeScript section and see an example, right? You'd see this:
![image](https://user-images.githubusercontent.com/6368074/53628077-5aaae700-3c0a-11e9-8702-83bc05c79b25.png)

Or am I not understanding you correctly?

Reply:
I must have missed the link 🤷‍♂️